src/Component/Filter/RangeNumericFilter.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Grid\Filter;
+
+use Sylius\Component\Grid\Data\DataSourceInterface;
+use Sylius\Component\Grid\Filtering\FilterInterface;
+
+final class RangeNumericFilter implements FilterInterface
+{
+    public const DEFAULT_SCALE = 0;
```

Comment:
I wonder which default scale would be sensible.

@Sylius/development-team wdyt?

SF numeric form type uses `3`.

Reply:
I think 0 is the most sensible. I don't think filtering with decimal precision is more common than with whole numbers.